src/api/java/com/minecolonies/api/tileentities/TileEntityRack.java

```diff
@@ -132,7 +133,7 @@ public boolean hasSimilarStack(@NotNull final ItemStack stack)
         {
             for (final ResourceLocation tag : stack.getItem().getTags())
             {
-                if (storage.getItemStack().getItem().getTags().contains(tag))
```

Comment:
Why are our tags excluded here?

Reply:
because we got some "all including tags" like "exclusion lists for crafters" where a lot of random items are matched together.

Reply:
Make that a config then. Cause there are a lot of tags which have that property potentially.

Reply:
mod exclusion config?